transhumandesign/kag-base
# Fix bad snapshot error that was caused by archer changing class on tent or other respawn points

## Status

(pick one)

- **READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description

When player changes class from archer to archer in tent or any other respawn, there might be a bad sync, that would cause a bad snapshot, or sometimes bad delta spam that would crash the server. Here i replaced Sync with Command to prevent this from happening.

## Steps to Test or Reproduce

Quickly press E and pick archer class couple of times, you will see errors in console.
